common/djangoapps/terrain/stubs/http.py

```diff
@@ -217,6 +218,8 @@ def _format_msg(self, format_str, *args):
         `format_str` is a string with old-style Python format escaping;
         `args` is an array of values to fill into the string.
         """
+        if not args:
+            format_str = urllib.unquote(format_str)
```

Comment:
Thanks @muzaffaryousaf.  I did miss that.  However, even with that, I'm still curious about my questions, even if we decide not to change any of the code.